pkg/scheduler/flavorassigner/flavorassigner_test.go

```diff
@@ -2172,30 +1791,24 @@ func TestAssignFlavors(t *testing.T) {
 					Request(corev1.ResourceCPU, "9").
 					Obj(),
 			},
-			clusterQueue: cache.ClusterQueue{
-				Cohort: &cache.Cohort{
-					Usage: resources.FlavorResourceQuantitiesFlat{
-						{Flavor: "one", Resource: corev1.ResourceCPU}: 2000,
-					}.Unflatten(),
-					RequestableResources: resources.FlavorResourceQuantitiesFlat{
-						{Flavor: "one", Resource: corev1.ResourceCPU}:  10000,
-						{Flavor: "one", Resource: corev1.ResourcePods}: 10,
-					}.Unflatten(),
+			clusterQueue: utiltesting.MakeClusterQueue("test-clusterqueue").
+				ResourceGroup(
+					utiltesting.MakeFlavorQuotas("one").
+						Resource(corev1.ResourcePods, "10").
+						ResourceQuotaWrapper(corev1.ResourceCPU).NominalQuota("10").LendingLimit("0").Append().
+						FlavorQuotas,
+				).Cohort("test-cohort").ClusterQueue,
+			clusterQueueUsage: resources.FlavorResourceQuantitiesFlat{
+				{Flavor: "one", Resource: corev1.ResourceCPU}: 2_000,
+			},
+			cohortResources: &cohortResources{
+				usage: resources.FlavorResourceQuantitiesFlat{
+					{Flavor: "one", Resource: corev1.ResourceCPU}: 2_000,
+				},
+				requestableResources: resources.FlavorResourceQuantitiesFlat{
+					{Flavor: "one", Resource: corev1.ResourceCPU}:  2_000,
```

Comment:
It looks like this was `10_000` before

Reply:
I'm pretty sure this test was added when the lendingLimit feature was added. So we are probably missing some coverage on it now. @kerthcet do you recall what was the intention of the test and can you send a PR to fix it?

Reply:
I'll take a look later. Thanks for reminding, almost missed it.

Reply:
I couldn't find the case, can anyone point that out？

Reply:
![image](https://github.com/kubernetes-sigs/kueue/assets/1299064/ced2d2aa-9993-4e56-b5f2-de10ffe77044)